mypy/checker.py

```diff
@@ -2599,7 +2599,8 @@ def check_simple_assignment(self, lvalue_type: Optional[Type], rvalue: Expressio
             return rvalue_type
 
     def check_member_assignment(self, instance_type: Type, attribute_type: Type,
-                                rvalue: Expression, context: Context) -> Tuple[Type, Type, bool]:
+                                rvalue: Expression, context: MemberExpr
```

Comment:
I would rename `context` to `lvalue`. Context is typically something that is used mostly for error line/column.